ruby/files_and_serialization/project_event_manager.md

```diff
@@ -892,11 +884,11 @@ We really want to capture the first name and last name of each legislator.
 Instead of outputting each raw legislator we want to print only their first
 name and last name. We will need to complete the following steps:
 
-* For each zip code, iterate over the array of legislators.
-* For each legislator, we want to find the representative's name.
-* Add the name to a new collection of names.
+- For each zip code, iterate over the array of legislators.
+- For each legislator, we want to find the representative's name.
+- Add the name to a new collection of names.
 
-To do this, we can use the [map](https://docs.ruby-lang.org/en/3.2/Array.html#method-i-map) function built into ruby. It works just like `.each` but returns a new array of the data we want to include.
+To do this, we can use the built-in [`Array#map` method](https://docs.ruby-lang.org/en/3.2/Array.html#method-i-map). It works just like `.each` but returns a new array of the data we want to include.
```

Comment:
```suggestion
To do this, we can use the built-in [Array#map method](https://docs.ruby-lang.org/en/3.2/Array.html#method-i-map). It works just like `.each` but returns a new array of the data we want to include.
```
Having links where we have only one of the words in a code block (or vice versa) often renders not very well (looks like two links at a glance):
![image](https://github.com/TheOdinProject/curriculum/assets/122839503/c43226bc-43f3-4473-8c46-5a3f0cca3676)

Compared to
![image](https://github.com/TheOdinProject/curriculum/assets/122839503/66d93bf2-bbcb-4719-af1e-b9a9d8cb7ed8)